### PR TITLE
Feature/highlighted hoop

### PR DESCRIPTION
**Pull request by Oskmey**
Added so that the hoop the player should pass i highlighted

### CODE REVIEW

File: crocket/src/main/java/com/crocket/view/LevelView.java [hunk 1]
```diff
@@ -130,6 +130,12 @@ private void getHoopTexture(DrawableEntity entity) {
             default:
                 throw new IllegalArgumentException("Invalid rotation");
         }
+
+        if(entity.isActive()) {
```
Comment (Pontus-Wikstrom) [inline crocket/src/main/java/com/crocket/view/LevelView.java:134]: add blankspace

**Review by Pontus-Wikstrom — APPROVED** [review]
Fix comment, otherwise LGTM

File: crocket/src/main/java/com/crocket/model/EntityDrawableVisitor.java [hunk 2]
```diff
@@ -37,12 +38,19 @@ private static EntityType getEntityTypeFromClass(Class<?> type) {
...
 
     private void addDrawableEntity(Entity entity, int rotation) {
+        boolean active = false;
         int xPosition = (int) entity.getxPosition();
         int yPosition = (int) entity.getyPosition();
         int width = entity.getWidth();
         int height = entity.getHeight();
+        IModel model = Model.getInstance();
+        Player activePlayer = model.getActivePlayer();
+        Entity entity2 = activePlayer.getActiveTarget();
```
Comment (filipkauffeldt) [inline crocket/src/main/java/com/crocket/model/EntityDrawableVisitor.java:48]: I would suggest renaming this to something more descriptive. Maybe `activeTarget`

File: crocket/src/main/java/com/crocket/model/entity/Hoop.java [hunk 3]
```diff
@@ -40,6 +40,9 @@ public void collideWithBall(Ball ball) {
         else if (collidedLeft(ball)) {
             Direction direction = CollisionHandler.collidedDirection(ball, leftHitbox);
             CollisionHandler.reflect(ball, direction);
+        } else if (passedThrough(ball)) {
+            PassTargetEvent event = new PassTargetEvent(ball, this);
```
Comment (filipkauffeldt) [inline crocket/src/main/java/com/crocket/model/entity/Hoop.java:44]: I would suggest removing this and only checking passedThrough once

**Review by filipkauffeldt — APPROVED** [review]

**Pull request merged**